src/components/Dropdown/Dropdown.tsx

```diff
@@ -89,6 +89,26 @@ export function Dropdown(props: PropsWithChildren<DropdownProps>): JSX.Element {
     } else if (e.key === 'ArrowUp') {
       updateFocusedItem(focusedIndex - 1);
     }
+    if (e.key === 'Tab' && !e.shiftKey) {
```

Comment:
we probably want these ifs to be else ifs

Reply:
good point